src/Catty/FormulaEditor/Form/FormItem.swift

```diff
@@ -47,13 +47,15 @@ class FormTextFieldItem: FormItem {
     var placeholder: String?
     var typeAction: ((String) -> Void)?
     var returnAction: (() -> Void)?
+    var becomeFirstResponder = false
 
-    init(typeAction: ((String) -> Void)? = nil, returnAction: (() -> Void)? = nil, placeholder: String?) {
+    init(typeAction: ((String) -> Void)? = nil, returnAction: (() -> Void)? = nil, placeholder: String?, becomeFirstResponder: Bool) {
```

Comment:
I would just call the variable and init parameter `focus` instead of `becomeFirstResponder`